Amazon.QLDB.Driver/Amazon.QLDB.Driver.csproj

```diff
@@ -23,6 +23,7 @@
     <AssemblyOriginatorKeyFile>..\public.snk</AssemblyOriginatorKeyFile>
     <ReleaseVersion>1.1.0</ReleaseVersion>
     <PackageIcon>product-icon_AWS_Quantum_125_squid-ink.png</PackageIcon>
+    <LangVersion>latest</LangVersion>
```

Comment:
Any reason we are using the latest lang version? Are we using any new features of the latest version?

Reply:
Yeah it was to get `await foreach (IIonValue value in result)` working in `AsyncBufferedResult::BufferResultAsync(IAsyncResult result)`.

Reply:
We should specify the exact C# version that supports that feature.

I also see

> Choosing a language version newer than the default can cause hard to diagnose compile-time and runtime errors.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

Can we update the change log and specify this change so we keep track of this?

Reply:
Done.